README.md

```diff
@@ -1,8 +1,9 @@
 # Introduction 
-Roslyn Diagnostic Analyzers are customized compiler errors providing real-time feedback to C# developers.  Each Analyzer optionally includes an automatic Code Fixer.  Microsoft (and other organizations) offer many analyzers, but the market is nascent.
+Roslyn Diagnostic Analyzers are customized compiler errors providing real-time feedback to C# developers. Many Analyzers include an automatic Code Fixer. While Microsoft (and other organizations) offer many analyzers, the market is nascent. Moreover, many tools offer a rich set of rules, but lack the shift-left integration that Roslyn achieves.
```

Comment:
I'm not sure how many people understand "shift-left", wouldn't is be better to mention something like "while you type feedback"